ui/js/controllers/logviewer.js

```diff
@@ -199,14 +199,6 @@ logViewerApp.controller('LogviewerCtrl', [
                     }
                 };
 
-                // get the revision and linkify it
-                ThResultSetModel.getResultSet($scope.repoName, job.result_set_id).then(function(data){
-                    var revision = data.data.revision;
-                    $scope.logProperties.push({label: "Revision", value: revision});
-                    $scope.logRevisionFilterUrl = $scope.urlBasePath +
-                        "#/jobs?repo=" + $scope.repoName + "&revision=" +
-                        revision;
-                });
```

Comment:
I think we actually still need most of this, it's just the `$scope.logRevisionFilterUrl = $scope.urlBasePath +
                        "#/jobs?repo=" + $scope.repoName + "&revision=" +
                        revision;` part which we can remove. In particular, I think appending to `$scope.logProperties` is essential.